.github/workflows/release.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
   release:
     needs: [lint, unit_test]
-    runs-on: ubuntu-24.04
+    runs-on: ubuntu-22.04
```

Comment:
any specific reason to use this version?

Reply:
Enables CLI binary compatibility with 22.04 ubuntu. It is still compatible with 24.04. 

If we build CLI on 24.04, users won't be able to install CLI binary on ubuntu 22.04

Reply:
```
ubuntu@ip-172-31-29-94:~$ /home/ubuntu/bin/avalanche --version
avalanche version 1.8.9-rc4
ubuntu@ip-172-31-29-94:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.5 LTS
Release:        22.04
Codename:       jammy

```

Reply:
```
ubuntu@ip-172-31-16-118:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 24.04.2 LTS
Release:        24.04
Codename:       noble
ubuntu@ip-172-31-16-118:~$ /home/ubuntu/bin/avalanche --version
avalanche version 1.8.9-rc4
```